index.html

```diff
@@ -60,13 +60,164 @@ <h3>I can help you build a product , feature or website Look through some of my
         
         </section>
 
+        <div class="body2">
```

Comment:
- [ ] Let's use `<section>` instead of the `div` tag for the works section to improve the structure and semantics.